app/controllers/orders_controller.rb

```diff
@@ -48,28 +80,89 @@ def new
   def update
     @order = Order.find(session[:order_id]) #session is persistent, the cookie has the session information.
     @order.update(order_param[:order]) # when you get a request, here is my information for right now.
+
     reduce_inventory(@order)
-    if @order.save
+
+    zip = params["order"]["zip"]
+    city = params["order"]["city"]
+    state = params["order"]["state"]
+    # message = ""
+    message = valid_location(zip, city, state)
+
+
+    if @order.save && message == ""
       redirect_to order_path
```

Comment:
I have no idea what any of this refers to. Please explain. Thanks.

Thomas Murphy
On Jun 7, 2016 4:32 PM, "Jennie Buechner" <notifications@github.com> wrote:

> In app/controllers/orders_controller.rb
> <https://github.com/vconklin/betsy/pull/1#discussion_r66171027>:
>
> >      reduce_inventory(@order)
> > -    if @order.save
> > +
> > +    zip = params["order"]["zip"]
> > +    city = params["order"]["city"]
> > +    state = params["order"]["state"]
> > +    # message = ""
> > +    message = valid_location(zip, city, state)
> > +
> > +
> > +    if @order.save && message == ""
> >        redirect_to order_path
>
> Rails has a handy helper method you could use here:
> message.blank?
>
> It would match if message were " " or nil, etc.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vconklin/betsy/pull/1/files/d404571ad98034b0f9de6d01b688165e61307f78#r66171027>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAZjcVbwAJnxzrDFo1T-ZGkmGhog2f_6ks5qJf-LgaJpZM4Ik99H>
> .
>